### PR TITLE
[#213] Add Amazon S3 docs

### PR DESCRIPTION
**Pull request by vitorbaptista**

### CODE REVIEW

File: docs/getting_started/s3.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+1. [Create a bucket on S3][howto-s3bucket] to hold your data
+    * Create the bucket on the `us-west-2` region. It's a [current limitation][s3-region-bug] of goodtables.io that we're working to fix.
+1. [Create a new IAM user][howto-iamuser]. This user will be used by goodtables.io to read your bucket.
+    * Make sure you take note of the AWS Access Key ID, AWS Secret Access Key, and the User ARN.
+1. Go to your [bucket's overview page][bucket-overview], click on the `Permissions` tab, and find the `Bucket Policy` link. We need the permissions:
+    * _s3:ListBucket_: To list the bucket's contents
+    * _s3:GetObject_: To read the bucket's files
+    * _s3:GetBucketPolicy_, _s3:PutBucketPolicy_, and _s3:PutBucketNotification_: To set up the AWS Lambda functions that notifies goodtables.io when a new file is added.
+    * _s3:GetBucketLocation_: TODO: Why we need this?
```
Comment (vitorbaptista) [inline docs/getting_started/s3.md]: @amercader I don't know why we need this permission. Could you explain?

Comment (amercader) [reply docs/getting_started/s3.md]: I'm pretty sure it was for the Lambda function to be able to access the bucket

Comment (amercader) [reply docs/getting_started/s3.md]: Now that I think about it, we are attaching the policy automatically ourselves so I'm not sure if you need to add this step on the docs.

Here's where it's done:

https://github.com/frictionlessdata/goodtables.io/blob/master/goodtablesio/integrations/s3/utils/bucket.py#L26

Comment (vitorbaptista) [reply docs/getting_started/s3.md]: @amercader As I'm suggesting creating a new IAM user to control the permissions of goodtables, this user needs to have permissions to change the bucket policy. The way to do this is via the bucket policy itself. Probably the user doesn't need to add all permissions as I've done in the example, but will need to add some of those. Do you know which?

**Pull request merged**